test/dummy/bin/generate

```diff
@@ -0,0 +1,3 @@
+#!/usr/bin/env bash
+
+protoc --twirp_ruby_out=proto/ --ruby_out=proto/ -I proto/ proto/api.proto
```

Comment:
We should extract this into a rake task provided by `rails-twirp` since every project that uses the gem will need something like this

Reply:
Good call!